build-system/tasks/clean.js

```diff
@@ -42,7 +42,7 @@ async function clean() {
     pathsToDelete.push('**/node_modules', '!node_modules');
   }
   const deletedPaths = await del(pathsToDelete, {
-    expandDirectories: true,
+    expandDirectories: false,
```

Comment:
Can we add a TODO to restore this to `true`?

Reply:
Never mind, this was merged before my comment posted. Will look for the underlying bug before restoring.